packages/app-page-builder/src/admin/views/Pages/Table/index.tsx

```diff
@@ -16,9 +23,17 @@ import { PageListWithConfig } from "~/admin/config/pages";
 
 const View = () => {
     const { currentFolderId } = useNavigateFolder();
+    const { getCurrentLocale } = useI18N();
+    const { tenant } = useTenancy();
+
+    const autoSaveId = useMemo(() => {
+        const localeCode = getCurrentLocale("content");
+        const applicationId = "pb:page";
+        return generateAutoSaveId(tenant, localeCode, applicationId);
+    }, [getCurrentLocale, tenant]);
```

Comment:
This code is very repetitive, and the only thing that changes is the `applicationId`. Let's expose a hook on the `SplitView` component, like `SplitView.useLayoutId`, and then use that hook to generate the `id` by passing an application id:

```tsx
const MyView = () => {
  // If CMS....
  const { model } = useModel();
  const layoutId = SplitView.useLayoutId(`cms:${model.modelId}`);

  // If PB...
  const layoutId = SplitView.useLayoutId("pb:page");

  return <SplitView layoutId={layoutId}>{/*.....*/}</SplitView>;
};
```

Also, the `autoSaveId` is very vague in the context of Webiny, and is a very specific prop for the underlying library. What does it mean `autoSaveId`, what are you auto saving? It's unclear. I suggest we change this to `layoutId`. SplitView has a certain layout, and that's its `id`. We can then do various things with this `layoutId`, doesn't have to be related to saving at all.